app/views/cas_users/show_history.html.erb

```diff
@@ -0,0 +1,39 @@
+<% content_for :page_title, "User History: #{@cas_user.cas_directory_id}" %>
+
+<h1><%= content_for :page_title %></h1>
+
+<%= form_tag '', method: 'get' do %>
+  Find events from the past
+  <%= number_field_tag :days, @days, size: 4, min: 1 %>
+  days.
+  <%= submit_tag 'Go', class: 'btn btn-notice' %>
```

Comment:
@peichman-umd The "Go" button does have any indication that it is a button. Should we replace 'btn-notice' calss with 'btn-primary'. Optionally, we can also include 'btn-sm' if we need to make it small.